sp800-63b/sec6_lifecycle.md

```diff
@@ -12,21 +12,23 @@ Throughout the online identity lifecycle, CSPs SHALL maintain a record of all au
 
-At IAL 2 and above, identifying information is associated with the online identity and the subscriber has undergone an identity proofing process as described in SP 800-63A. As a result, authenticators at the same AAL as the desired IAL SHALL be bound to the account. For example, if the subscriber has successfully completed proofing at IAL 2, AAL 2 or 3 authenticators are appropriate to bind to the IAL 2 identity.  As above, the availability of additional authenticators provides backup methods of authentication if an authenticator is damaged, lost, or stolen.
+At IAL 2 and above, identifying information is associated with the online identity and the subscriber has undergone an identity proofing process as described in SP 800-63A. As a result, authenticators at the same AAL as the desired IAL SHALL be bound to the account. For example, if the subscriber has successfully completed proofing at IAL 2, AAL 2 or 3 authenticators are appropriate to bind to the IAL 2 identity. While a CSP MAY bind an AAL 1 authenticator to an IAL 2 identity, if the subscriber is authenticated at AAL 1, the CSP SHALL NOT expose personal information, even if self-asserted, to the subscriber. As above, the availability of additional authenticators provides backup methods of authentication if an authenticator is damaged, lost, or stolen.
 
-Enrollment and binding MAY be broken up into a number of separate physical encounters or electronic transactions. (Two electronic transactions are considered to be separate if they are not part of the same protected session.) In these cases, the following methods SHALL be used to ensure that the same party acts as Applicant throughout the processes:
+Enrollment and binding MAY be broken up into a number of separate physical encounters or electronic transactions. Two electronic transactions are considered to be separate if they are not part of the same protected session. In these cases, the following methods SHALL be used to ensure that the same party acts as applicant throughout the processes:
 
 1. For remote transactions:
-	2. The applicant SHALL identify himself/herself in each new transaction by presenting a temporary secret which was established during a prior transaction or encounter, or sent to the Applicant’s phone number, email address, or postal address of record.
-	3. Long-term authenticator secrets SHALL only be issued to the Applicant within a protected session.
+	2. The applicant SHALL identify themself in each new transaction by presenting a temporary secret which was established during a prior transaction or encounter, or sent to the applicant’s phone number, email address, or postal address of record.
```

Comment:
Singular "they" reflexive form is "themselves" not "themself" in formal contexts: https://en.oxforddictionaries.com/usage/themselves-or-themself